src/app/auth/login/login.component.scss

```diff
@@ -51,13 +56,15 @@
 }
 
 .sub-title {
-  margin-top: 10px;
-  font-size: 24px;
+  @include fonts-style(24px, 600, 12px, 1px, $accent-state-text);
+
```

Comment:
Please separate font styles from component ones in the common _typography.scss file

Reply:
I'll do it in a separate pull request